### ISSUE TITLE
build in support for mutation testing

### ISSUE THREAD

**Issue by Waldbillig-Thomas** [angular/angular-cli#19440]
🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅

Oh hi there! 😄 

🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅

# 🚀 feature request

### Relevant Package
@angular/cli

### Description
I was just wondering if you would consider adding mutation testing to the basic generated app. It really helps to learn about creating meaningful unit tets (by far more important than simple coverage)

### Describe the solution you'd like
I know that [stryker](https://stryker-mutator.io/) has support for angular. If it could be added by default to the basic generated app, I think it would help a lot of people improving their skills (as well as mine) for writing unit tests

### Describe alternatives you've considered
 Have you considered any alternative solutions or workarounds?

**Comment by clydin** [comment]
Thank you for the feature request.
However, adding additional testing frameworks is not currently on the roadmap of Angular CLI.  Also, where possible the Angular CLI tries to create an initial project that does not diverge from the tools used by Google itself.

If interested in using Stryker, the Stryker project currently provides guides for integrating into several of the major web frameworks.  The Angular guide can be found here: https://stryker-mutator.io/docs/stryker/guides/angular
The Angular Team has also in the past provided assistance to the Stryker team to support a well-functioning integration for users ([related issue](https://github.com/angular/angular-cli/issues/8744)).

**Comment by Waldbillig-Thomas** [comment]
Hi there, thanks for your reply.

Just a small thing: adding Stryker would not be really be adding a testing
framework, since all it does it evaluating your existing tests by mutating
the code being tested. It will show you which tests do nothing, what code
might be in the coverage but is not actually tested, and what cases your
existing unit tests might have missed.

Tldr;
Keep up the good work and thanks,
Thomas
